.github/workflows/credcheck.yml

```diff
@@ -0,0 +1,58 @@
+name: credentials-sanity-check
+
+on:
+  workflow_dispatch:
+
+jobs:
+  check-credentials:
+    permissions:
+      id-token: write
+      contents: write
+    runs-on: ubuntu-latest
+    name: Check release credentials are valid
+    steps:
+      - name: Checkout custom actions
+        uses: actions/checkout@v3
+        with:
+          clean: false
+      - name: Use Node 18
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+      - name: Get vault secrets
+        id: secrets
+        uses: SonarSource/vault-action-wrapper@dc8fe04652687f7278f3ecd27905967836bab0eb # tag=2.7.4-1
+        with:
+          secrets: |
+            development/artifactory/token/{REPO_OWNER_NAME_DASH}-private-reader access_token | ARTIFACTORY_ACCESS_TOKEN;
```

Comment:
We could get the PAT from Vault here with the proper key (and also update the matching post-release action).